src/main/java/edu/iis/powp/adapter/PlotterCommandAdapter.java

```diff
@@ -0,0 +1,27 @@
+package edu.iis.powp.adapter;
+
+import edu.iis.client.plottermagic.IPlotter;
+import edu.iis.powp.builder.ComplexCommandBuilder;
+import edu.iis.powp.command.PlotterCommand;
+
+public class PlotterCommandAdapter implements IPlotter, PlotterCommand {
+    ComplexCommandBuilder builder = new ComplexCommandBuilder();
+
+    @Override
+    public void setPosition(int i, int i1) {
+        builder.addSetPositionCommand(i, i1);
+    }
+
+    @Override
+    public void drawTo(int i, int i1) {
+        builder.addDrawToCommand(i, i1);
+    }
+
+    @Override
+    public void execute(IPlotter plotter) {
+        PlotterCommand command = builder.build();
```

Comment:
Mixing two responsibilities, generating command and executing it. It would be simpler to build and return command.